src/GitHub.Exports.Reactive/Extensions/ConnectionManagerExtensions.cs

```diff
@@ -12,33 +15,40 @@ public static class ConnectionManagerExtensions
         public static IObservable<bool> IsLoggedIn(this IConnectionManager cm, IRepositoryHosts hosts)
```

Comment:
These methods should be changed to return `async Task<T>`  but I didn't want to have to update all the call-sites in the PR.